spec/support/face_book_omniauth.rb

```diff
@@ -0,0 +1,14 @@
+module FaceBookOmniauth
+  def self.omniauth_info
+    omni_config = OmniAuth.config
+    omni_config.test_mode = true
+    omni_config.mock_auth[:facebook] = OmniAuth::AuthHash.new(
+      'provider' => 'facebook', 'uid' => '1234567890',
```

Comment:
use VCR to stub real data or create a `json` fixture
would be much easier to manage

Reply:
I'm personally doesn't like VCR because it requires running test with real credentials to collect response. So on my projects we are stubbing responses with json files.

BUT Misha you can try to use VCR here just to learn how to use it.